client/src/components/Homepage.js

```diff
@@ -18,40 +19,91 @@ function Homepage({ searchText }) {
       });
   }
 
-  useEffect(loadFeed, [feedURL, searchText]);
+  function toggleSortOrder() {
+    setSortOrder(sortOrder === 'ASC' ? 'DESC' : 'ASC');
+  }
```

Comment:
Should the sort toggle be a drop down or should it be a button like you have it now? If we make it a drop down then there is potential for more sorting techniques and also it eliminates accidental clicks since it requires two

Reply:
I'm gonna keep it as a button for now until we need the additional features. However, I do agree a drop down would work well if we want additional sort modes.